import/llm-records/src/generate.records.ts

```
@@ -0,0 +1,49 @@
+
+Return only the JSON data, no other text or comments.`
+
+  const llm = new llmHandler(model, apiKey)
+  const response = await llm.handleMessage('', prompt)
+  const responseText = response.content as string
+
+  if (debug) {
+    console.dir(responseText, { depth: null })
+  }
+
+  const jsonRegex = new RegExp(`\`\`\`json\\n([\\s\\S]*?)\\n\`\`\``)
```

Biome

[error] Use a regular expression literal instead of the RegExp constructor.

Regular expression literals avoid some escaping required in a string literal, and are easier to analyze statically.
Safe fix: Use a literal notation instead.


(lint/complexity/useRegexLiterals)